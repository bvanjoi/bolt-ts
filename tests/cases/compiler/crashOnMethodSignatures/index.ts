// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/crashOnMethodSignatures.ts`, Apache-2.0 License

class A {
  a(completed: () => any): void;
  //~^ ERROR: Function implementation is missing or not immediately following the declaration.
}
