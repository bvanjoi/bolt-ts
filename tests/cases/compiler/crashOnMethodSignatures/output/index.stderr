  × Function implementation is missing or not immediately following the
  │ declaration.
   ╭─[../../tests/cases/compiler/crashOnMethodSignatures/index.ts:4:3]
 4 │ a
   · ─
   ╰────
